### PR TITLE
EBP: User Preferences Page styles

### PR DESCRIPTION
**Pull request by SammyIsConfused**
##### Checklist

- [x] the [contributor license agreement][] is signed
- [x] commit message follows [commit guidelines][]
- [x] screenshots are included showing significant UI changes

##### Description of change
#1983 
- Save and Change password buttons padding and shadow fix
- First and last name alignment
- Timezone/Language section alignment
- Checkbox padding and alignment
- Header size and font aligns with other new stuff (I.E contribution page)
- Uses MUI theme colours
- Added padding to expanded dropdowns when they are at the bottom of a page
- Change dividers to use MuiDivider styles
- border radius for the watched collections box
Peek 2020-09-16 17-24
image
image

[contributor license agreement]: https://www.apereo.org/node/676
[commit guidelines]: https://chris.beams.io/posts/git-commit/

### CODE REVIEW

File: Source/Plugins/Core/com.equella.core/resources/web/sass/legacy.scss [hunk 1]
```diff
@@ -5219,3 +5228,78 @@ button:focus {
...
+    margin-left: $singleMargin;
+  }
+
+  #ed_em {
+    width: 495px;
+  }
+
+  .itemdefs_container {
+    padding: $singleMargin;
+    border-radius: 3px;
```
Comment (edalex-ian) [inline Source/Plugins/Core/com.equella.core/resources/web/sass/legacy.scss:5267]: Could make sense to pull this 3px out into a variable, and then you can also re-use in those other places which has this. One place was the buttons, however looking above I now see that's at 4px. Shouldn't these match? If so, should it be 4 or 3? (I guess that highlights the use of putting it in a variable.)

**Review by edalex-ian — CHANGES_REQUESTED** [review]
Looks nice and tidy @SammyIsConfused !

Just need to push a couple of things into variables to keep it DRY for easy maintenance and consistency.

**Review by mrblippy — COMMENTED** [review]
It's much tidier now that those extra variables are being used :+1:

File: Source/Plugins/Core/com.equella.core/resources/web/sass/legacy.scss [hunk 2]
```diff
@@ -1579,11 +1592,9 @@ Alerts
 *****************************************************************************/
 
 .alert {
+  border-radius: 4px;
```
Comment (edalex-ian) [inline Source/Plugins/Core/com.equella.core/resources/web/sass/legacy.scss:1595]: Should this be the new `curvedCorners`?

**Review by edalex-ian — APPROVED** [review]
Looks good, you just missed one `curvedCorners`. GTG after that's fixed.

**Pull request merged**